### PR TITLE
Move threats from common to observation section in forms 

### PR DESCRIPTION
**Pull request by onyxvd**
- Remove "Threats" from common section in forms
 - Add "Threats" field in all forms
 - Rename threats/findings field to "Findings" in herptiles, mammals and invertebrates

## Form_Birds
* threats -> main_threats, moved bottom
## Form_CBM
* threats -> main_threats, moved bottom
## Form_Ciconia
* threats -> main_threats, moved bottom
## Form_Plants
* threats -> main_threats, moved bottom
* remove threatsPlants -> plants_threats
## Form_Herptiles
* threats -> main_threats, moved bottom
* findings -> herptiles_findings
* threatsHerptiles is deprecated
## Form_Mammals
* threats -> main_threats, moved bottom
* findings -> mammals_findings
* threatsMammals is deprecated
## Form_Invertebrates
* threats -> main_threats, moved bottom
* findings -> invertebrates_findings
* threatsInvertebrates is deprecated

### CODE REVIEW

File: client/views/monitorings/herptiles.html [hunk 1]
```diff
@@ -55,9 +55,9 @@ <h1 class="page-header">
                  model="monitoringDetailController.data.habitat"></field>
         </div>
         <div class="col-md-12 col-lg-12">
-          <field name="threatsHerptiles" type="multiple-choice" nomenclature="herptiles_danger_observation"
-                 label="{{'HERPTILES_DETAIL_THREATS' | translate}}"
-                 model="monitoringDetailController.data.threatsHerptiles"></field>
+          <field name="findings" type="multiple-choice" nomenclature="herptiles_danger_observation"
```
Comment (groupsky) [inline client/views/monitorings/herptiles.html]: @GPopgeorgiev maybe the nomenclature should also be renamed?

File: client/views/monitorings/herptiles.html [hunk 2]
```diff
@@ -140,6 +140,14 @@ <h1 class="page-header">
...
+    <div class="col-md-12">
+      <field name="threats" type="multiple-choice" nomenclature="main_threats"
+             label="{{'COMMON_FORM_THREATS' | translate}}"
+             model="monitoringDetailController.data.threats"></field>
+    </div>
+  </div>
+
   <div class="row">
     <div class="col-md-12">
       <field name="speciesNotes" type="memo"
```
Comment (groupsky) [inline client/views/monitorings/herptiles.html]: `speciesNote` and `threats` seems to be in every form, maybe extract as partial template?

File: server/migrations/20190514080324-rename-threats-findings-columns.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    await queryInterface.renameColumn('FormHerptiles', 'threatsHerptilesBg', 'findingsBg')
+    await queryInterface.renameColumn('FormHerptiles', 'threatsHerptilesEn', 'findingsEn')
+    await queryInterface.renameColumn('FormMammals', 'threatsMammalsBg', 'findingsBg')
+    await queryInterface.renameColumn('FormMammals', 'threatsMammalsEn', 'findingsEn')
+    await queryInterface.renameColumn('FormInvertebrates', 'threatsInvertebratesBg', 'findingsBg')
+    await queryInterface.renameColumn('FormInvertebrates', 'threatsInvertebratesEn', 'findingsEn')
```
Comment (groupsky) [inline server/migrations/20190514080324-rename-threats-findings-columns.js]: @GPopgeorgiev is this correct :point_up:? The existing field `threats` will not be touched, just moved to be populated on each record (mobile) and bottom (web)

**Comment by groupsky** [comment]
Pending approval from @GPopgeorgiev before merging in master

**Comment by groupsky** [comment]
@onyxvd instead of rename, add the new field. will remove the old one later on, when everyone is using the new app version

**Comment by groupsky** [comment]
Use new nomenclature for findings

**Comment by groupsky** [comment]
Extract common footer with `threats` and `speciesNotes`

**Comment by kalkov** [comment]
Extract: 
```html
<field name="threats" type="multiple-choice" nomenclature="main_threats"
             label="{{'COMMON_FORM_THREATS' | translate}}"
             model="monitoringDetailController.data.threats"></field>
```
into a partial. 

`speciesNotes` extraction in partial can be done in separate PR.

**Pull request merged**